src/Workleap.DomainEventPropagation.Subscription/Extensions/ServiceCollectionEventPropagationExtensions.cs

```diff
@@ -23,18 +23,20 @@ public static IEventPropagationSubscriberBuilder AddEventPropagationSubscriber(t
         return new EventPropagationSubscriberBuilder(services);
     }
 
-    public static IEndpointRouteBuilder AddEventPropagationEndpoints(this IEndpointRouteBuilder builder)
+    public static RouteHandlerBuilder AddEventPropagationEndpoint(this IEndpointRouteBuilder builder)
```

Comment:
Should we return using `=>` lamda?

Reply:
Good point we could! I just realized I should return the builder in the WithAuthorization method